### DISCUSSION TITLE
[Support]: Frigate+: How to best handle having to add boxes of the same things on every image I review

### DISCUSSION THREAD

**Discussion by d-steve** [blakeblackshear/frigate#11597] [General Support] [answered]
### Describe the problem you are having

Considering moving to Frigate+, decided to start reviewing images to get a feel for the workflow.

One of my cameras has several parked cars in it that I expect to be in nearly every image. They don't create events because I use object masks and/or required_zones, as needed, but they do cause me to have to add boxes for them for every image I review. Here's an example:

![snapshot](https://github.com/blakeblackshear/frigate/assets/76608172/e7ab56e3-b3b4-4a70-9fac-fc26f3b68dab)

It appears to be necessary to mark some/all of thel five extra cars in this image, for every image I upload. Am I missing something, or is there going to be a way to do reduce the required effort, for example, any of the following?

1) automatically adds boxes for all identified objects Guessing this would need to be a reprocessing of the image.
2) allow manually adding boxes that mask areas of the image. I get that you wouldn't get the benefit of positively identifying the objects each time, but this reduces effort such that the mask doesn't have to be as precise and doesn't have to be labeled vs precisely adding a new box and accurately labeling.

(2) would only be needed if (1) doesn't exist or in environments where (1) isn't effective.

Thanks. For *everything*

### Version

0.13.2-6476F8A

### Frigate config file
```shell
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Dahua IPC-T54IR-AS S3

**Comment by NickM-27** [comment, marked as answer]
there is a pinned feature request for automatic labeling. For now I would recommend becoming familiar with the many keyboard shortcuts that speed up the time it takes to annotate images

**Reply by d-steve** [reply]
Ah, I had already searched/looked in discussions. Didn't know where to find pinned feature requests but just found it under "Issues". Too much manual effort for now, think I'll wait for that. Thanks!
